CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.0.0 - May 5, 2020
```

Comment:
Update the date? :)
Also forgot to add 1.0.3 here but it's in the git release versions 🤷